users/api/transport.go

```diff
@@ -39,13 +39,20 @@ func MakeHandler(svc users.Service, tracer opentracing.Tracer) http.Handler {
 
 	mux := bone.New()
 
-	mux.Post("/users", kithttp.NewServer(
+	mux.Post("/users/create", kithttp.NewServer(
 		kitot.TraceServer(tracer, "register")(registrationEndpoint(svc)),
 		decodeCredentials,
 		encodeResponse,
 		opts...,
 	))
 
+	mux.Post("/users", kithttp.NewServer(
+		kitot.TraceServer(tracer, "create_user")(createUserEndpoint(svc)),
```

Comment:
this looks reversed,  but we dont need this in this PR?

Reply:
Actually, I endeavor to replace `registrationEndpoint` with the new `createUserEndpoint`. Therefore, I used `/users` endpoint for `createUserEndpoint` and used `/users/create` for `registrationEndpoint`. However, it seems like this approach creates inconsistency about naming.